modules/installation-azure-service-principal.adoc

```diff
@@ -222,8 +221,7 @@ If you limit the service principal scope of the {product-title} installation pro
 [source,terminal]
 ----
 $ az role assignment create --role "User Access Administrator" \
-    --assignee-object-id $(az ad sp list --filter "appId eq '<appId>'" \
-       | jq '.[0].id' -r) <1>
+  --assignee-object-id $(az ad sp show --id <appId> --query objectId -o tsv) <1>
```

Comment:
--assignee-object-id $(az ad sp show --id <appId> --query Id -o tsv)

Reply:
@MayXuQQ 
Does the `--id` option not require a value like `<appId>`?

Reply:
checked with latest cli 
$ az version
{
  "azure-cli": "2.41.0",
  "azure-cli-core": "2.41.0",
  "azure-cli-telemetry": "1.0.8",
  "extensions": {}
}
**objectId** changed to **id**,  no **objectId** property 
$ az ad sp show --id 9102be60-9fa3-4c4e-90e6-60419fc23a22 | grep -i id
  "appId": "9102be60-9fa3-4c4e-90e6-60419fc23a22",
  "appOwnerOrganizationId": "6047c7e9-b2ad-488d-a54e-dc3f6be6a7ee",
  "applicationTemplateId": null,
  "id": "296bcdbf-bfc3-4edb-ac29-461928484712",
      "id": "3e2c7291-f75b-49c6-b8aa-dce1b61e4fd1",
  "tokenEncryptionKeyId": null,
    "verifiedPublisherId": null

Reply:
Great. Thanks for checking.